scripts/azure-pipelines/build.ps1

```diff
@@ -0,0 +1,48 @@
+﻿# Builds Mumble using the given environment and build script.
```

Comment:
"using the given" should maybe be formulated as an assumption/requirement paragraph.
Something like 
> "Builds Mumble..."
> 
> This script expects to be called... [in this kind of environment]

Reply:
Done.